code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -76,6 +76,16 @@
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
 	return
 
+/obj/item/weapon/storage/box/security
+
+/obj/item/weapon/storage/box/security/New()
+	..()
+	contents = list()

```

Comment:
Do not do this.

Do not clear a list by setting it to list()

This causes a major bug in that anything currently in that list will never get deleted, even if you call del() on it, and even if the round is restarted.

http://www.byond.com/forum/?post=1854520